Xamarin.Forms.Platform.iOS/CollectionView/UICollectionViewDelegator.cs

```diff
@@ -21,6 +21,20 @@ public UICollectionViewDelegator(ItemsViewLayout itemsViewLayout, ItemsViewContr
 
 		public override void WillDisplayCell(UICollectionView collectionView, UICollectionViewCell cell, NSIndexPath path)
 		{
+			switch (ItemsViewController.ItemsView.RemainingItemsThreshold)
+			{
+				case -1:
+					return;
+				case 0:
+					if (path.Row == ItemsViewController.ItemsViewSource.Count - 1)
```

Comment:
@hartez If I'm not mistaken, you were using `indexPath.Row` throughout `CollectionView` implementation, but I believe the correct property is `indexPath.Item` for collections. Here's a [discussion](https://stackoverflow.com/a/14765781/246392) on that. I left this to be `Row` for now, but we should consider switching to `Item`.